package.json

```diff
@@ -9,18 +9,19 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@fontsource/dm-sans": "^5.1.0",
```

Comment:
Actually we can directly use next/google fonts package already included in Nextjs. I'll update for the whole site once merged. Keep it for now.